pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -948,7 +948,10 @@ def test_parse_dates_custom_euro_format(all_parsers, kwargs):
             )
 
 
-@xfail_pyarrow
+# TODO: Depending on what tests are selected (i.e. pytest marks are selected)
+#  this test can xpass (2020-02-12: Azure MacOS 3.10 & GHA Linux Datamanager 3.8 xpass)
```

Comment:
1) year should be 2022, not 2020?
2) are you sure this isn't just xpassing bc there's a new pyarrow version?

Reply:
Good catch. Looks like it was a pyarrow version